release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -103,6 +103,21 @@ The Network Resources Injector that is deployed with the Operator is enhanced to
 
 For more information, see xref:../networking/hardware_networks/about-sriov.adoc#nw-sriov-hugepages_about-sriov[Huge pages resource injection for Downward API].
 
+[id="ocp-4-8-tracking-network-flows"]
+==== Tracking network flows
+
+{product-title} {product-version} adds support for sending the metadata about network flows on the pod network to a network flows collector. The following protocols are supported:
+
+* NetFlow
+
```

Comment:
Is the spacing in the list necessary? When looking at the list in the RN above, it doesn't look like there's a space.